test/integrations/destinations/algolia/dataDelivery/business.ts

```diff
@@ -235,16 +239,6 @@ export const testScenariosForV1API: ProxyV1TestData[] = [
                 statusCode: 500,
               },
             ],
-            statTags: {
```

Comment:
Are statTags not required in the response ? Generally statTags will be used for metric labels